test/helpers/port_list.ts

```diff
@@ -1,51 +1,36 @@
-const PORT = 9876
-const PORTAND40 = PORT + 40
-const PORTAND41 = PORT + 41
-const PORTAND42 = PORT + 42
-const PORTAND43 = PORT + 43
-const PORTAND44 = PORT + 44
-const PORTAND45 = PORT + 45
-const PORTAND46 = PORT + 46
-const PORTAND47 = PORT + 47
-const PORTAND48 = PORT + 48
-const PORTAND49 = PORT + 49
-const PORTAND50 = PORT + 50
-const PORTAND72 = PORT + 72
-const PORTAND103 = PORT + 103
-const PORTAND114 = PORT + 114
-const PORTAND115 = PORT + 115
-const PORTAND116 = PORT + 116
-const PORTAND117 = PORT + 117
-const PORTAND118 = PORT + 118
-const PORTAND119 = PORT + 119
-const PORTAND316 = PORT + 316
-const PORTAND326 = PORT + 326
-const PORTAND327 = PORT + 327
-const PORTAND400 = PORT + 400
+/**
+ * Method used to get ports for testing
+ * @param i Index to shift the ports by
+ * @returns
+ */
+export default function getPorts(i = 0) {
```

Comment:
trick to be sure ports doesn't overlap, each file must use a different `i` in order to work